organization-fields.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/*
+Plugin Name: Organization User Fields
+Description: Adds Department and Manager fields to the user profile.
+Version: 1.0
+Author: Gravity Flow
+Author URI: http://www.stevenhenty.com
+License: GPL-3.0+
+
+Copyright 2018 Steven Henty S.L.
+
+*** Instructions ***
+Use in merge tags e.g
+{user:manager} {user:department}
```

Comment:
Merge tag needs to be {user:manager_id} based on the user meta being saved.

Reply:
Fixed - thanks!